streams/readable-byte-streams/general.any.js

```diff
@@ -2064,15 +2064,46 @@ promise_test(() => {
   const reader = rs.getReader({ mode: 'byob' });
   const readPromise = reader.read(new Uint8Array(16));
   return pullCalledPromise.then(() => {
-    const cancelPromise = reader.cancel('meh');
-    resolvePull();
+    controller.close();
```

Comment:
After calling `cancel()`, `byobRequest.respond(0)` throws immediately (because we now invalidate the BYOB request). However, this test is trying to show that you can't call it *twice*. So I changed it to close the stream, rather than canceling it.

I've added a new test below this one for when the stream is canceled. 🙂